src/drive/store/connectedRouter.spec.js

```diff
@@ -3,10 +3,12 @@ import { connectStoreToHistory, createReducer } from './connectedRouter'
 import { hashHistory as history } from 'react-router'
 
 const routes = [
-  '/folder/:folderId/file/:fileId',
-  '/files/:folderId/file/:fileId',
   '/files/:folderId',
-  '/folder/:folderId'
+  '/folder/:folderId',
+  '/files/:folderId/file/:fileId',
+  '/folder/:folderId/file/:fileId',
+  'sharings/files/:fileId',
```

Comment:
Is it expected to have a leading / here ?

Reply:
Nope, good catch!